test-cypress/integration/ttyg/agent-list.spec.js

```diff
@@ -53,8 +53,10 @@ describe('TTYG agent list', () => {
         TTYGViewSteps.getAgents().should('have.length', 2);
         // Then Agent list filter should be set to All
         TTYGViewSteps.getSelectedAgentFilter().should('contain', 'starwars');
-        TTYGViewSteps.verifyRepositoryOptionNotExist('Fedx_repository');
-        TTYGViewSteps.verifyRepositoryOptionNotExist('Ontop_repository');
+        // TTYGViewSteps.getAgentFilter().click();
+        // TODO: Ask Boyan what are these lines for?
```

Comment:
These lines check that the list does not contain 'FedX' or 'Ontop' repositories. We have this requirement